habitat-baselines/habitat_baselines/config/experiments_hab3/hab3_single_robot.yaml

```diff
@@ -0,0 +1,40 @@
+# @package _global_
+
+defaults:
+  - /rearrange/rl_hierarchical
+  - override /habitat_baselines/rl/policy: hl_neural
+  - override /habitat_baselines/rl/policy/hierarchical_policy/defined_skills: oracle_skills
+  - override /hydra/output: path
+  - override /hydra/launcher: submitit_habitat
+  - _self_
+
+habitat_baselines:
+  video_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/video
+  tensorboard_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/tb
+  eval_ckpt_path_dir: ${hydra:sweep.dir}/${hydra:sweep.subdir}/checkpoints
+  checkpoint_folder: ${hydra:sweep.dir}/${hydra:sweep.subdir}/checkpoints
+  log_file: ${hydra:sweep.dir}/${hydra:sweep.subdir}/train.log
+  evaluate: False
+  writer_type: 'wb'
+  wb:
+    project_name: 'hab3'
+    entity: 'andrew-colab'
```

Comment:
Quick question! hab3 and andrew-colab here are all fixed right? We cannot change them here, otherwise it will not show up on wb?

Reply:
yup, for this projects, we plan to run all runs on this workspace. So we should keep this fixed.